app/controllers/followings_controller.rb

```diff
@@ -0,0 +1,26 @@
+class FollowingsController < ApplicationController
+  before_action :require_user
+
+  def create
+    @following = current_user.followings.build(follow_id: params[:follow_id])
```

Comment:
`build` and `new` are now synonymous, but `new` is generally used. In the past they used to be slightly different.